### ISSUE TITLE
Question: Which datasets was pretrained Point-SAM trained on?

### ISSUE THREAD

**Issue by SpeedyGonzales949** [zyc00/Point-SAM#9]
Hi! Thanks  for the awesome work! 

I saw you published the weights file on HugginFace. Which Datasets  did you use for this specific training ? Did you use every dataset mentioned in the paper ?

Thanks for the help!

**Comment by Jiayuan-Gu** [comment]
Yes, we used all the datasets mentioned in the paper: PartNet, PartNet-Mobility, Fusion360, ScanNet, and ShapeNet (pseudo labels).

**Comment by LinLin1031** [comment]
> Yes, we used all the datasets mentioned in the paper: PartNet, PartNet-Mobility, Fusion360, ScanNet, and ShapeNet (pseudo labels).

If I want to migrate Point-SAM's capabilities to other datasets, how should I train them? Can you provide the training code? @Jiayuan-Gu

**Comment by zyc00** [comment]
> > Yes, we used all the datasets mentioned in the paper: PartNet, PartNet-Mobility, Fusion360, ScanNet, and ShapeNet (pseudo labels).
> 
> If I want to migrate Point-SAM's capabilities to other datasets, how should I train them? In other words, how is the pre-trained model ‘model.safetensors’ trained? Can you provide the training code? @Jiayuan-Gu @zyc00

Thanks for your interest! The training code is released, please check our current main branch.
